src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientProvider.java

```diff
@@ -28,10 +29,14 @@ public BlobContainerClientProvider(
     public BlobContainerClient get(TargetStorageAccount targetStorageAccount, String containerName) {
         switch (targetStorageAccount) {
             case BULKSCAN:
+                Configuration config = new Configuration();
+                config.put(Configuration.PROPERTY_HTTPS_PROXY, "proxyout.reform.hmcts.net:8080");
+                config.put(Configuration.PROPERTY_HTTP_PROXY, "proxyout.reform.hmcts.net:8080");
```

Comment:
is this needed both here and in application main()?

Reply:
We should be able to remove all proxy config and test